components/math/multiplicationConnect/PlayerAndDice.tsx

```diff
@@ -71,7 +71,7 @@ const PlayerAndDice: FC<PlayerAndDiceProps> = ({ normalMode }) => {
             shake && "animate-shake"
           }`}
         >
-          <p className="text-sm">dice 1</p>
+          <p className="hidden text-sm sm:block">dice 1</p>
```

Comment:
I see you're working on mobile responsiveness.
Just something to keep in mind in case you didn't see in the tailwind docs.  Maybe for future consider using the normal styling for mobile and then md for the normal screens.

"Where this approach surprises people most often is that to style something for mobile, you need to use the unprefixed version of a utility, not the sm: prefixed version. Don’t think of sm: as meaning “on small screens”, think of it as “at the small breakpoint“.

Don't use sm: to target mobile devices"
Use unprefixed utilities to target mobile, and override them at larger breakpoints

Reply:
Yeah, I noticed! In this case I want the element to be hidden on mobile device and appear at the small breakpoint (640px)